language/vm/src/views.rs

```diff
@@ -477,7 +477,7 @@ impl<'a, T: ModuleAccess> FunctionDefinitionView<'a, T> {
     }
 
     pub fn is_public(&self) -> bool {
-        self.function_def.is_public()
+        matches!(self.function_def.visibility, Visibility::Public)
     }
```

Comment:
Switch this view to `pub fn visibility(&self) -> Visibility`

Reply:
I imagine you will have more places to clean after touching this, thats why hit request changes :)

Reply:
It will inch into the `move-model` land which I have less confidence in. I'll also request reviews from Wolfgang after the change.